aws/table_aws_ses_template.go

```diff
@@ -0,0 +1,187 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/ses"
+	"github.com/aws/aws-sdk-go-v2/service/ses/types"
+
+	sesv1 "github.com/aws/aws-sdk-go/service/ses"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableAwsSESTemplate(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_ses_template",
+		Description: "AWS SES Template",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("name"),
+			Hydrate:    getSESTemplate,
+			Tags:       map[string]string{"service": "ses", "action": "GetTemplate"},
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listSESTemplates,
+			Tags:    map[string]string{"service": "ses", "action": "ListTemplates"},
+		},
+		HydrateConfig: []plugin.HydrateConfig{
+			{
+				Func: getSESTemplate,
+				Tags: map[string]string{"service": "ses", "action": "GetTemplate"},
+			},
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(sesv1.EndpointsID),
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name of the template.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "subject_part",
+				Description: "The subject line of the email.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getSESTemplate,
+			},
+			{
+				Name:        "text_part",
+				Description: "The email body that will be visible to recipients whose email clients do not display HTML.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getSESTemplate,
+			},
+			{
+				Name:        "html_part",
+				Description: "The HTML body of the email.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getSESTemplate,
+			},
+			{
+				Name:        "created_timestamp",
+				Description: "The time and date the template was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "last_updated_timestamp",
+				Description: "The time and date the template was last updated.",
+				Type:        proto.ColumnType_TIMESTAMP,
```

Comment:
Do you think it would be better if we handle potential null values? What does the API return if the template has never been updated?